src/modules/jobs/models/tech-load.model.ts

```diff
@@ -0,0 +1,399 @@
+    }
+  }
+
+  /**
+   * @description - Called by constructor to generate a job. 
+   * @returns {JobData} - Returns an object which contains the entirety of job data required to fill out a fake ticket
+   */
+  private generateJob(): JobData {
+    // The following variables are generated prior to the output object, as they are passed into generator methods in order to generate data consistent throughout the ticket
+    const firstName = this.generateFirstName();
+    const lastName = this.generateLastName();
+    const streetAddress = this.generateStreetAddress();
+    const jobType = this.randOf(['Install', 'Repair', 'Helper', 'POTS', 'BSW']);
+    const transportType = jobType === 'POTS' || this.nextRand() < 0.5 ? this.randOf(['IP-CO', 'IP-RT', 'FTTN', 'FTTN-BP']) : this.randOf(['FTTP GPON', 'FTTP XGS-PON']);
+    const curJob: JobData = {
+      firstName: firstName,
+      lastName: lastName,
+      accountNumber: this.randDigits(7),
+      location: {
+        zip: this.randDigits(5),
+        ...this.generateCityLocation(streetAddress)
+      },
+      appointment: this.generateAppointment(),
+      email: this.generateEmail(firstName, lastName),
+      phone: this.randDigits(10),
+      history: this.generateHistory(),
+      jobType: jobType,
+      transportType: transportType,
+      facilities: this.generateFacilities(transportType, streetAddress),
+      services: this.generateServices(transportType)
+    };
+
+    return curJob;
+  }
+
+  /**
+   * @description Found online. Converts a string to a unique number for seed generation.
+   * @param {string} str The string to convert
+   * @returns {number} the string converted to a unique number
+   */
+  private convertStrToNum(str: string): number {
+    let hash = 0;
+    for (let i = 0; i < str.length; i++) {
+      hash = ((hash << 5) - hash) + str.charCodeAt(i); // essentially convert the number to binary, shift the bits over 5, and add the new number
+      hash |= 0; // converts the number to a 32bit integer
+    }
+    return hash;
+  }
+
+  /* * * * * * * * * * * * * * * * * * * * * * */
+  //          Seeded Methods
+  /* * * * * * * * * * * * * * * * * * * * * * */
+
+  /**
+   * @description Will generate the next number in the seed from the sequence, by incrementing the seed index and executing a prime number-based formula to produce a pseudorandom number.
+   * @returns {number} The next pseudorandom number produced from the seed, between 0 and 1 
+   */
+   private nextRand(): number {
+    this.seedIndex++;
+    let a = (this.seed + this.seedIndex) * 15485863; // primes and formula taken from https://en.wikipedia.org/wiki/Pseudorandom_number_generator#Implementation
+    return (a * a * a % 2038074743) / 2038074743;
+  }
+
+  /**
+   * @description Will return a random element of the given array based off of the global seeded random number generator
+   * @param {string[]} options An array of possible options to pseudo-randomly choose from 
```

Comment:
Doesn't match below

Reply:
Changed to T[]. I assume that's how you'd document the use of a generic?